README.md

```diff
@@ -15,33 +15,34 @@ This library is available as a package on NPM. To install into a project using N
 You can grab your [token](https://app.chameleon.io/setup/install) then require the lib like a standard Node.js module:
 
 ```javascript
-const chmln = require('@chamaeleonidae/chmln');
+const chameleon = require('@chamaeleonidae/chmln');
```

Comment:
I would revert some changes to the documentation, especially [this line](https://github.com/chamaeleonidae/chmln/pull/8/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R42)

Reply:
+1